openwisp-qa-check

```diff
@@ -117,7 +136,14 @@ runcheckmigrations() {
 }
 
 runflake8() {
-  flake8 && echo "SUCCESS: Flake8 check successful!" ||
+  EXCLUDE_PATTERN=".git,__pycache__,build,dist,*.egg-info"
+
+  if [ -n "$IGNORE_PATTERN" ]; then
+    EXCLUDE_PATTERN="$IGNORE_PATTERN"
+  fi
```

Comment:
This is incosistent, here we are overriding exclude pattern with ignore pattern completely